docs/docs/flink-writes.md

```diff
@@ -75,7 +75,7 @@ Iceberg supports `UPSERT` based on the primary key when writing data into v2 tab
     ```
 
 !!! info
-    OVERWRITE and UPSERT can't be set together. In UPSERT mode, if the table is partitioned, the partition fields should be included in equality fields.
+    OVERWRITE and UPSERT can't be set together. In UPSERT mode, if the table is partitioned, the source columns of partition fields should be included in equality fields.
```

Comment:
I don't understand this change. (I don't understand the sentence).
Could you please help me understand?
Thanks,
Peter

Reply:
Basically descriptions of the following codes

https://github.com/apache/iceberg/blob/91dff9886e6e6c494f6a970129f4c08487c98a0d/flink/v2.0/flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java#L537-L545

Reply:
Could you please add an example like this to the doc?
Now, I understand what `source columns of` means. But I needed the example to understand the exact meaning.

Thanks,
Peter